backend/docker-compose.yml

```diff
@@ -6,35 +6,38 @@ services:
       context: .
       target: demo
       args:
-        SECRET_KEY: ${SECRET_KEY:?}
+        SECRET_KEY: tmp_key
     image: spellbook-backend
-    expose:
-      - 8000
+    ports:
+      - 8000:80
```

Comment:
I think that the container is still using port 8000

Reply:
There is nginx running directly on the pod for ease of use with kubernetes. So the nginx on the container is indeed port 80 (it passes along the gunicon at 8000).

Reply:
Thanks for your explanations